roles/nova-data/templates/etc/serverspec/kvm_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe file('/var/lib/libvirt/images') do
+  it { should be_mode 711 }
+end
+
+describe file('/dev/kvm') do
+  it { should exist }
+  it { should be_mode 660 }
+end 
+
+describe file('/etc/libvirt/libvirtd.conf') do
+  file_contents = ['unix_sock_group = "libvirtd"',
+                   'unix_sock_ro_perms = "0777"',
+                   'unix_sock_rw_perms = "0770"']
+  file_contents.each do |file_line|
+    it { should contain file_line }
+  end
+end
+
+describe file('/etc/libvirt/libvirtd.conf') do
+  it { should contain "^listen_tls = 0" }
+  it { should contain "^listen_tcp = 1" }
```

Comment:
That kind of thing usually do to it being a different rule in the techspec (read: ITCS104). When we label them in the future with the yaml techspec definitions, they will be two separate checks.